docs/source/discord/resources/channel/methods/get_channel_invites.rst

```diff
@@ -8,12 +8,19 @@ Example
 
 .. code:: c
 
-    struct discord_invite **ret;
-   
-    discord_get_channel_invites(client, msg->channel_id, &ret);
-    
-    if (ret)
-      for (int i = 0; ret[i]; ++i)
-        printf("%s\n", ret[i]->code);
+   void done_get_channel_invites(struct discord *client, void *data, const struct discord_invites *ret)
+   {
+     if (ret)
```

Comment:
Please check the [guide](https://gist.github.com/lcsmuller/a5f2b205c3871888656b86825db90187#example-3) for the new method of iterating our built-in array types.